### PR TITLE
multiprocessing

### PR DESCRIPTION
**Pull request by ghost**
Greetings,

Sorry for the confusion,
Reorganization of src code resulted in deletion.
What exactly is required to proceed with the PR?

Thanks!

### CODE REVIEW

File: fractal.py [hunk 1]
```diff
@@ -124,15 +135,41 @@ def generate_fractal(model, c=None, size=pair_reader(int)(DEFAULT_SIZE),
...
+    args = [(width, height, side, deltax, deltay, cx, cy, zoom, row, func, depth, c)]
+    p = pool.apply_async(generate_process, args)
+    procs.append(p)
+
+  # Collect results from worker procs
+  for row in range(height):
+    img[row, :] = procs[row].get()
+
+  # Print the timer
+  print 'Time taken:', time.time()-start
```
Comment (danilobellini) [inline fractal.py]: There's no print statement in Python 3. There's no problem on keeping the "Time taken" print, but it need parentheses and the `print_function` also imported from `__future__` to work on both versions of Python. As logging doesn't seem like a problem, you can print the "Amount of procs" before the time taken, it should be useful as well.

File: fractal.py [hunk 2]
```diff
@@ -124,15 +135,41 @@ def generate_fractal(model, c=None, size=pair_reader(int)(DEFAULT_SIZE),
...
-    for row in range(height)
-  ])
-
+  
+  # Start a timer
+  start = time.time()
+
+  # Create a pool of workers
+  numProcs = multiprocessing.cpu_count()
+  pool = multiprocessing.Pool(numProcs)
```
Comment (danilobellini) [inline fractal.py]: Should be `num_procs`, as the code never uses lowerCamelCase. Not needed as a variable unless you print such information afterwards avoiding the private `pool._processes`.

**Comment by danilobellini** [comment]
For now don't worry too much about the tests failing on Travis CI, I already know the `--pre` matplotlib version has something weird and I'm looking for a fast, elegant and easily maintainable way to solve it that won't give me a timeout problem. You can locally run the tests with py.test on both Python 2 and 3, that should be enough. I'm trying to find a way to make the tests work again on Travis CI, soon this part will be fixed and hopefully the GitHub auto-merge will yield a test-passing code. =)

**Comment by ghost** [comment]
Hello,

I plan implement all requirements soon.
Work in progress.

Thanks,
coast

**Comment by ghost** [comment]
Hello, 
The print function cleanup is complete.
Thanks,
coast

**Comment by ghost** [comment]
Hello,

What are your thoughts on merging this request?

Thanks,
Coast

**Comment by danilobellini** [comment]
Thanks for the PR!

I hadn't merge because Travis times out with multiprocessing, but this seem to be a Travis issue as the code is passing on my computer, so I'm merging it now =)

**Comment by danilobellini** [comment]
GitHub complains when I press the merge button on a phone, so I'll rebase it at home (better than merging, IMHO)

**Comment by danilobellini** [comment]
It's already there, rebased. Thanks! =)

**Pull request closed without merging**